config/jobs/kubernetes-sigs/cluster-api-provider-gcp/cluster-api-provider-gcp-ci.yaml

```diff
@@ -121,3 +121,47 @@ periodics:
   annotations:
     testgrid-dashboards: sig-cluster-lifecycle-cluster-api-provider-gcp
     testgrid-tab-name: e2e-tests-stable
+- name: ci-cluster-api-provider-gcp-make-conformance-stable-k8s-ci-artifacts
+  interval: 3h
+  labels:
+    preset-service-account: "true"
+    preset-bazel-scratch-dir: "true"
+    preset-bazel-remote-cache-enabled: "true"
+    preset-dind-enabled: "true"
+    preset-kind-volume-mounts: "true"
+  spec:
+    containers:
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20191012-482f444-master
+        env:
+          - name: "CAPI_BRANCH"
+            value: "stable"
+          - name: "CABPK_BRANCH"
+            value: "stable"
+        args:
+          - "--repo=k8s.io/kubernetes=master"
+          - "--repo=sigs.k8s.io/cluster-api-provider-gcp=master"
+          - "--repo=sigs.k8s.io/cluster-api=master"
+          - "--repo=sigs.k8s.io/image-builder=master"
+          - "--repo=sigs.k8s.io/kind=master"
```

Comment:
We should probably target a release of kind instead of master here.

Reply:
i copied the stable job. i don't think we even need the kind line here. will update all the jobs in a later PR